index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
```

Comment:
since it's the number of ms since 1 January 1970 00:00:00 UTC you'd have to make sure to stagger the `Date.now` calls.

I don't think it's necessary anymore since like you said, we can just `dehydrate` & `rehydrate` the `lastId` as well.